library/Helpers/CanValidateDateTime.php

```diff
@@ -59,6 +61,17 @@ private function isDateTimeParsable(array $info): bool
         return $info['error_count'] === 0 && $info['warning_count'] === 0;
     }
 
+    /**
+     * Validates if the given string is a valid DateInterval type.
+     * 
+     * @param string $age
+     * @return bool
+     */
+    private function isDateIntervalType(string $age): bool
+    {
+        return in_array($age, array_keys(get_object_vars((new DateInterval('P1Y')))));
+    }
```

Comment:
I know this seems weird, but i wanted to use those DateInterval vars. what do you guys think about it? How do we can improve this validation?